src/App.jsx

```diff
@@ -14,70 +16,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? (
+            <>
+              {`${selectedGood} is selected`}
+              <button
+                onClick={() => setGoods('')}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </>
+          )
+          : ('No goods selected')
+        }
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = selectedGood === good;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                data-cy="Good"
+                className={cn({ 'has-background-success-light': isSelected })}
+                key={good}
+              >
+                <td>
+                  <button
+                    onClick={() => (
+                      setGoods(!isSelected && good)
+                    )}
+                    data-cy={isSelected ? 'RemoveButton' : 'AddButton'}
+                    type="button"
+                    className={`button ${isSelected && 'is-info'}`}
```

Comment:
better to use the classnames package to avoid "false" class
<img width="456" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/70220667/2cf59150-a4c0-499d-b490-0bb37965ada6">